src/lamplib/src/genny/tasks/compile.py

```diff
@@ -50,9 +50,9 @@ def cmake(
     # using both.
     cmake_prefix_paths = [
         os.path.join(
-            toolchain_info.toolchain_dir, f"installed/x64-{toolchain_info.triplet_os}-dynamic",
```

Comment:
These hard-coded x64 are lousy in the vcpkg code also. I made similar changes there.